test/functorch/test_control_flow.py

```diff
@@ -73,6 +75,168 @@ def f(x, pred, pred2):
 
         self.assertEqual(result_false_true, torch.cos(x))
 
+    def test_cond_functionalized(self):
+        def true_fn(x):
+            y = x.sin()
+            y.add_(4)
+            return x.sin().max() + y.sum()
+
+        def false_fn(x):
+            return x.cos().min()
+
+        def f(x):
+            pred = x.shape[0] == 1
+            return cond(pred, true_fn, false_fn, [x])
+
+        example_inputs = (torch.ones(4, 5),)
+        functional_f = functionalize(f)
+        self.assertEqual(functional_f(*example_inputs), f(*example_inputs))
+
+        graph_module = make_fx(functionalize(f))(*example_inputs)
+        self.assertEqual(graph_module(*example_inputs), f(*example_inputs))
+
+        all_ops_in_true_branch = []
+        for node in graph_module.true_graph_0.graph.nodes:
+            if node.op == "call_function":
+                all_ops_in_true_branch.append(node.target)
+
+        self.assertFalse(torch.ops.aten.add_.Tensor in all_ops_in_true_branch)
+        self.assertFalse(torch.ops.aten.add_.Scalar in all_ops_in_true_branch)
+        self.assertFalse(any([op._schema.is_mutable for op in all_ops_in_true_branch]))
```

Comment:
nit: first two lines are redundant with the third line (self.assertFalse(any([op._schema.is_mutable for op in all_ops_in_true_branch])))